pages/studentPortal/web/React/assignments/template.tsx

```diff
@@ -39,7 +39,7 @@ const React2 = ({ incompleteStage, submittedStage, completedStage }) => {
     let submittedAssignment = assignments.filter(
       (assignment) => assignment.submission_link
     );
-    if (submittedAssignment) {
+    if (submittedAssignment.length > 0) {
```

Comment:
This query does work...and if I put in the following div, returns the information that I want: 
 ```
<div>
        {assignments.map((assignment) => (
          <div>{assignment.submission_link}</div>
        ))}
      </div>
```
I am unclear, however, why this query returns [object, object] and the field __typename and therefore does not allow me to print out data or data.user_assignment_submissions etc.

wondering if there is a better way?